[FIX] We were not verifying the configuration value `ignore_ssl` at the moment of `socketio` connection. #212
